doc/api/http.md

```diff
@@ -917,6 +917,32 @@ Example:
 var contentType = response.getHeader('content-type');
 ```
 
+### response.getHeaders([namesOnly])
```

Comment:
Wouldn't it be better if we `getHeaderNames`? We would avoid polymorphic return types as well.